lib/brie.js

```diff
@@ -38,7 +38,7 @@ const brie = module.exports = (function () {
     Object.assign(functionalCriteria, optCriteria);
     features = optFeatures;
     overrides = optOverrides;
-    Object.assign(data, optContext);
+    data = optContext
```

Comment:
```suggestion
    data = Object.assign({}, optContext)
```
Might be better?

Reply:
We did talk about this offline; since `optContext` is not addressed any further, and since we don't set values on data hereafter, the mutation prevention is not essential.